User-Stories/PO-PM-User-Stories/PO-user-stories.md

```diff
@@ -0,0 +1,60 @@
+#Product Owner (PO) / Project Manager (PM) user stories
+We assume this role as a principal role compared to a developer. Thus this player has the right to task the developer (agent) and influences the priotization of the agents work. Typically such a role is a team lead or project manager or in modern words a product owner. For ease of read, we will title the role PO, konwing that this simplification will not cover all details associated with the different role models. But from a compliance perspective the models can be treated as equivalent.
+
+##Responsibilites, tasks and role
+The ```primary goal``` of this role is to deliver a software solution while minimizing all related risks. This may comprise security, export controls or the incompliance with obligations resulting from the application of open source components. 
+
+Thus this role will be ```accountable``` for:
+* Execution of all reuired tasks to provide a risk minimized software solutions
+* Arrange and provide all relevnat tasks to cope with resulting obligations
+* Completeness of files/data/code to be assessed and delivered within the scope of this project
+
+This typically results in the follwing ```tasks```:
+* Ensure code is scanned on a regular basis - optimally as part of the CI/CD-chain
```

Comment:
Which code is meant - do you require a snippet scanner? 
I think the coplaince tool chain shall be integrated in the CI CD chain per default

Reply:
Actually the idea is to ensure source code of the solution which is subject of declarations is analyzed on a regular base; ideally during/before each build. This requirement is independent from the scope of the analysis. Whether it is only composition or provenance should be users decision.

Reply:
Now I understand - perhaps you can add a clarification that not necessarily a snippet scanner is meant

Reply:
added a line to clarify this